applications/launchpad/gui-react/public/index.html

```diff
@@ -25,6 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <script src="http://localhost:8097"></script>
```

Comment:
I guess it's a React DevTools artifact. In my opinion it can stay if we add `@TODO` tag above, ie:

```
<!-- @TODO @REMOVE / React DevTools / remove before release -->
```

Then, we can search by `@TODO` or `@REMOVE`, and check in a few weeks what needs to be _fixed_.

Reply:
Done.